free-4.md

```diff
@@ -0,0 +1,92 @@
+# Free project 4 - Encoding Decoder
+
+## The user and a language
+This section describes who the project would serve and why a language might be a
+good way to meet their needs.
+
+
+### What's the need?
+_What need is met by your idea? Who are you helping? What is that person's
+experience like now? What would their experience be like if you could help 
+them?_
+
+This project would serve weird people (me) who want the ability to transform text into different encodings. If a person were writing a message in Morse Code and realized that they wanted to see what it looked like in Braille, this language would be able to meet their needs.
+
+The idea meets the needs of people who enjoy encodings but don't want to take the time translating between them. It would be especially useful for events like [puzzlehunts](https://en.wikipedia.org/wiki/Puzzlehunt) in which participants solve puzzles. Some of these puzzles involve multiple ciphers which the average person can't translate on the fly. With this language, the user would only have to input the encoded text as well as the language of encoding/decoding and the user would understand what the encoded text was saying.
```

Comment:
This is an interesting idea, although my first instinct is it seems really really hard to do.